### PR TITLE
13 struct refactor play

### PR DESCRIPTION
**Pull request by zacdemi**
Refactored Play route into separate components.

### CODE REVIEW

File: src/components/modules/Play/Play.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      let devKeys = Object.keys(devData);
+      let prodKeys = Object.keys(prodData);
+
+      devKeys.forEach(devKey => {
+        let repoEntry = devData[devKey];
+        // search look for substring and return its position of its first occurance
+        if (repoEntry.name.search("project__") == 0) {
+          projectsData.push({
+            title: repoEntry.name,
+            description: repoEntry.description + " | " + repoEntry.html_url,
```
Comment (dev-launchers-sandbox) [inline src/components/modules/Play/Play.js]: Could we get the URL to be a clickable link on a separate line, maybe a tag with the inner value of "GitHub Repo" or "View Source On GitHub" or something of the sort?

Comment (zacdemi) [reply src/components/modules/Play/Play.js]: For Sure. I can conditionally render it in the EntryCard if its prop size is 'large' (until we get attachments). Unless you think there's a different way. Another alternative is to have the picture and title link directly to the Github page and put the link to run the game at the top of the repository.

Comment (zacdemi) [reply src/components/modules/Play/Play.js]: Sure. I created a temporary github attachment image to be include in the header.

**Pull request merged**